source/MaterialXGenShader/Syntax.h

```diff
@@ -132,12 +139,13 @@ class TypeSyntax
 protected:
     /// Protected constructor
     TypeSyntax(const string& name, const string& defaultValue, const string& uniformDefaultValue, 
-        const string& typeDefStatement, const vector<string>& members);
+        const string& typeAlias, const string& typeDefinition, const vector<string>& members);
 
     string _name;                // type name
     string _defaultValue;        // default value syntax
     string _uniformDefaultValue; // default value syntax when assigned to uniforms
-    string _typeDefStatement;    // custom typedef statement if needed in source code
+    string _typeAlias;           // type alias if needed in source code
+    string _typeDefinition;      // custom type definition if needed in source code
```

Comment:
Made a separation between simple type aliases and new type definitions (like structs etc.).